python/dgl/nn/pytorch/glob.py

```diff
@@ -47,30 +92,76 @@ def forward(self, graph, feat):
 
 
 class AvgPooling(nn.Module):
-    r"""Apply average pooling over the nodes in the graph.
+    r"""
+
+    Description
+    -----------
+    Apply average pooling over the nodes in a graph.
 
     .. math::
         r^{(i)} = \frac{1}{N_i}\sum_{k=1}^{N_i} x^{(i)}_k
+
+    Notes
+    -----
+        Input: Could be one graph, or a batch of graphs. If using a batch of graphs,
+        make sure nodes in all graphs have the same feature size, and concatenate
+        nodes' feature together as the input.
+
+    Examples
+    --------
+    The following example uses PyTorch backend.
+
+    >>> import dgl
+    >>> import torch as th
+    >>> from dgl.nn.pytorch.glob import AvgPooling
+    >>>
+    >>> g1 = dgl.DGLGraph()
+    >>> g1.add_nodes(2)
+    >>> g1_node_feats = th.ones(2,5)
+    >>>
+    >>> g2 = dgl.DGLGraph()
+    >>> g2.add_nodes(3)
+    >>> g2_node_feats = th.ones(3,5)
+    >>>
+    >>> avgpool = AvgPooling()
+
+    Case 1: Input single graph
+
+    >>> avgpool(g1, g1_node_feats)
+        tensor([[1., 1., 1., 1., 1.]])
+
+    Case 2: Input a batch of graphs
+
+    Build a batch of DGL graphs and concatenate all graphs' note features into one tensor.
+
+    >>> batch_g = dgl.batch([g1, g2])
+    >>> batch_f = th.cat([g1_node_feats, g2_node_feats])
+    >>>
+    >>> avgpool(batch_g, batch_f)
+        tensor([[1., 1., 1., 1., 1.],
+                [1., 1., 1., 1., 1.]])
     """
     def __init__(self):
         super(AvgPooling, self).__init__()
 
     def forward(self, graph, feat):
-        r"""Compute average pooling.
+        r"""
+
+        Compute average pooling.
 
         Parameters
         ----------
-        graph : DGLGraph
-            The graph.
+        graph : DGLGraph or a batch of DGLGraphs
```

Comment:
Why mentioning a batch of DGLGraphs? As far as I know it only accepts a single DGLGraph (batched or not batched).